data/legacy-libraries.json

```diff
@@ -0,0 +1,232 @@
+[
```

Comment:
There are 2 missing legacy libraries that should also be included in this file:

- MathJax: https://moodledev.io/general/community/credits/thirdpartylibs#mathjax
- mp3player: https://moodledev.io/general/community/credits/thirdpartylibs#mp3player

Reply:
Done